lcls_tools/common/data_analysis/archiver.py

```diff
@@ -52,14 +64,24 @@ def timestamp(self) -> datetime:
 
     @property
     def is_valid(self) -> bool:
+        """Determines if the value corresponding to the ArchiverValue object is valid."""
         return self.severity is not None and self.severity != EPICS_INVALID_VAL
 
 
 class ArchiveDataHandler:
```

Comment:
Class is implemented great and the methods for getting timestamps, values, etc are clear.